Makefile

```diff
@@ -100,11 +100,11 @@ cljs-browser-test: node_modules package-lock.json
 cljs-node-test: node_modules package-lock.json
 	npx shadow-cljs release node-test
 
-nodejs-test: out/flureenjs.js
+nodejs-test: out/fluree-node-sdk.js
 	cd test/nodejs && npm install && node --experimental-vm-modules node_modules/jest/bin/jest.js
 
-browser-test: out/flureedb.js
-	cd test/browser && npm install && npm ci
+browser-test: out/fluree-browser-sdk.js
+	cd test/browser && npm install && CI=true npm test
```

Comment:
Do we always want CI=true here? What does that do in `npm test`?

Reply:
Thanks for the docs link. They suggest: For non-CI environments, you can pass the `--watchAll=false` flag to disable test-watching. That might be more appropriate since this make target isn't always necessarily running in CI.

Reply:
I did try that but couldn't get it to work - it would still enter "watch" mode when executed via make:

`cd test/browser && npm install && npm test --watchAll=false`

It works when I do it manually, though. Maybe I'm just holding it wrong though? But the CI env var does the trick.